<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="backgroundView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameSignupLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>289</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:solid;
border-width:1px;
border-color:black;
border-radius: 10px;</string>
   </property>
   <property name="placeholderText">
    <string> username ...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordSignupLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:solid;
border-width:1px;
border-color:black;
border-radius: 10px;</string>
   </property>
   <property name="placeholderText">
    <string> password ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="signupLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 28pt &quot;Teko SemiBold&quot;;
color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(76, 255, 76, 255)); 
background-color: rgb(0, 0, 0);
border:solid;
border-width:1px;
border-color:black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="signinLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 28pt &quot;Teko SemiBold&quot;;
color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 76, 76, 255)); 
background-color: rgb(0, 0, 0);
border:solid;
border-width:1px;
border-color:black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>SIGN IN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordSigninLineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:solid;
border-width:1px;
border-color:black;
border-radius: 10px;</string>
   </property>
   <property name="placeholderText">
    <string> password ...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameSigninLineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>289</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:solid;
border-width:1px;
border-color:black;
border-radius: 10px;</string>
   </property>
   <property name="placeholderText">
    <string> username ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 16pt &quot;Teko SemiBold&quot;;
color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);
border:solid;
border-width:1px;
border-color:black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 16pt &quot;Teko SemiBold&quot;;
color: rgb(255, 0, 0);
background-color: rgb(0, 0, 0);
border:solid;
border-width:1px;
border-color:black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
